components/src/components/index.ts

```diff
@@ -38,17 +34,13 @@ export default {
   simpledatetime,
   simpleday,
   simpleemail,
-  simplefieldset,
   simplefile,
-  simpleheading,
   simplenumber,
   simplepanel,
-  simpleparagraph,
   simplephonenumber,
   simpleradios,
   simpleselect,
   simpletabs,
   simpletextarea,
   simpletextfield,
-  simpletime
```

Comment:
I'm a bit concerned about removing these components from the export package - have we verified that forms using these components already do not break?